consentor-archetype/src/test/resources/projets/basic/archetype.properties

```diff
@@ -0,0 +1,6 @@
+package=com.example.curity.extensions
+version=1.0.0-SNAPSHOT
+groupId=com.example.curity
+artifactId=example-curity-hello-world-consentor
+pluginName=HelloWorldConsentor
+curitySdkVersion=5.0.0
```

Comment:
new lines should be added. Without, things like `cat`, produce harder-to-use or wrong results